XamlControlsGallery/ControlPages/TabViewPage.xaml

```diff
@@ -211,6 +212,48 @@
             </local:ControlExample.Substitutions>
         </local:ControlExample>
 
+        <local:ControlExample HeaderText="The close button can be persistent or only visible on hover">
+            <local:ControlExample.Example>
+                <muxc:TabView x:Name="TabView31" SelectedIndex="0" Margin="-12" IsAddTabButtonVisible="False" MinHeight="475">
```

Comment:
Is `x:Name` needed here? If not, please remove. If so, please rename to "TabView4" (or something similar).

Reply:
I've adjusted the name. Since the samples below did not need the name, and we would have had collisions, I have removed the `x:Name` for those samples